### DISCUSSION TITLE
resetSelectedRows action resets selection to initialState. I would need resetting to empty selection.

### DISCUSSION THREAD

**Discussion by wuarmin** [TanStack/table#4146] [Q&A]
Hey,

the discussion-title says it. My use-case is to be able to preselect a row via `initialState`, but when user paginates or filters the table and the rows change, I need to reset the `selectedRowIds` to an empty row selection. At the moment the reset of `selectedRowIds` results in `initialState.selectedRowIds`

see:
https://github.com/TanStack/table/blob/b77ed829917ab3254be48bff2aac7602b9133982/src/plugin-hooks/useRowSelect.js#L106

WDYT? 
best regards

**Comment by KevinVandy** [comment]
Pass true to resetRowSection(true)

**Reply by wuarmin** [reply]
Oh I forgot to say, that I'm using v7. `resetRowSelection` is available in v8. We need the same feature in v7.
